openzwave-sys/src/lib.rs

```diff
@@ -6,6 +6,16 @@ pub mod options;
 pub mod notification;
 pub mod value_classes;
 
+use std::path::PathBuf;
+
+pub fn get_default_config_path() -> PathBuf {
+    let installed_path = env!("CARGO_MANIFEST_DIR");
```

Comment:
This is set at build time, not at run time :)

but yes, if we don't build with cargo we'll likely have an issue ;)